phpstan-baseline.neon

```diff
@@ -15,6 +15,11 @@ parameters:
 			count: 1
 			path: src/Document/BaseDocumentManager.php
 
+		-
+			message: "#^Method Sonata\\\\Doctrine\\\\Model\\\\BaseManager\\:\\:create\\(\\) should return T of object but returns mixed\\.$#"
```

Comment:
Do you know why we have this new error ?

Reply:
Becouse `return new $this->class();`. 
phpStan do not know what class is `$this->class`. W must say what is is:
```php
/**
     * @var string
     * @phpstan-var class-string<T> // add this line
     */
    protected $class;
```

Reply:
@core23 Do you have time ti fix it?